requirements.txt

```diff
@@ -1 +1,2 @@
 requests>=1.0
+retrying==1.3.3
```

Comment:
Thanks, I'll have to replace this with retrying>=1.3.3 (need the jitter and the six update for py3 compat)